lib/platform/gitlab/__snapshots__/index.spec.ts.snap

```diff
@@ -1600,7 +1600,7 @@ Array [
       "user-agent": "https://github.com/renovatebot/renovate",
     },
     "method": "GET",
-    "url": "https://gitlab.com/api/v4/projects/some%2Frepo/repository/files/file.json?ref=HEAD",
+    "url": "https://gitlab.com/api/v4/projects/some%2Frepo/repository/files/dir%2Ffile.json?ref=HEAD",
```

Comment:
To make `getJsonFile` tests look more uniform, I've intentionally added `dir/` prefix here